src/spicedb-common/lang/dslang.ts

```diff
@@ -650,6 +647,7 @@ export default function registerDSLanguage(monaco: any) {
       { token: 'identifier.permission', foreground: '000000' },
       { token: 'identifier.relation', foreground: '000000' },
     ],
+    colors: {},
```

Comment:
I'm not sure if this changed in a recent version of Monaco that we unintentionally picked up, but if you don't define this, Monaco errors out because it's undefined. This fixes things.